### ISSUE TITLE
Admin Theme - Test with popular 3rd party modules

### ISSUE THREAD

**Issue by rtpHarry** [OrchardCMS/Orchard#7267]
Towards the end of this process we should install the top 10 most popular modules on the gallery and check them over to see if there are any major issues with the new theme.

Other fringe cases should crop up when it graduates to dev branch.

**Comment by ifle** [comment]
I'm new in orchard. What  is top 10 most popular modules? Thanks

**Comment by rtpHarry** [comment]
There is [a module gallery](http://gallery.orchardproject.net/Packages/Modules) and by default it is sorted by Most Popular.

I don't know if these are necessarily the best of the best and still recommended for a new Orchard installation but for the testing purposes my logic is that if they have been installed the most then we need to make sure that they are work with the new admin theme as they are likely still amongst the most used modules out there in existing sites.

**Comment by ifle** [comment]
I know about module gallery. Top 5 most popular modules published 2-4 years ago. The question are these module still compatibility and relevant to latest version of orchard. In Umbraco gallery there is option to mark each module with what version it is tested.

**Comment by rtpHarry** [comment]
Yeah the gallery website needs work, there are a lot of plans floating around to enhance it, including have some kind of user contributed "works with vX.X" module but the work only ever gets done if somebody volunteers to code it and nobody has stepped up yet. The answer is, unfortunately, that the vast majority of modules need attention, either because they are years out of date or because of the last 1.10 update where the modules moved up to target a new .net version and we moved over to nuget packages.

**Comment by Skrypt** [comment]
Drupal has it's own git server so they can bundle modules automatically for all the contrib modules. I think that we need to figure out something that will automate things like that. Most modules made by contributors are on different Github, Bitbucket, Codeplex accounts which makes it harder for automation of module packages. I think that ultimately we should have a module gallery that retrieves repositories with something like [libgit2sharp](https://github.com/libgit2/libgit2sharp) and that makes use of the git Tags and/or Releases. Right now since we don't own any of those contributed repositories we can't create releases/tags and test those modules which should be handled by a CI server like appveyor. There's a whole workflow that has yet to be defined before making the Orchard Gallery better.
